### ISSUE TITLE
Repeated, persistent notifications with bluetooth connected watch

### ISSUE THREAD

**Issue by pandaM3CH4** [TeamNewPipe/NewPipe#1574]
Hello, NewPipe development team!

I use an Android 8.0.0 phone (a Samsung Galaxy S8) with a Bluetooth smart watch (a Garmin Fenix 5). My issue concerns the background player notification.

When I start background play, the NewPipe notification is repeatedly, quickly and persistently sent to my watch (which buzzes and flashes like crazy). Sometimes, toggling the pause/play will temporarily stop the repeated notifications.

This issue does not occur with similar notification-tray players such as Google Play Music. (The issue is not an emergency; I stopped NewPipe notifications on the watch.)

Thought you should know! Thank you for a great app :smiley:

- [x] I carefully read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md) and agree to them.
- [X] I checked if the issue/feature exists in the latest version.
- [n/a] I did use the [incredible bugreport to markdown converter](https://teamnewpipe.github.io/CrashReportToMarkdown/) to paste bug reports.

**Comment by theScrabi** [comment]
I see, well we had some problems with smarthwatches before, however since no one of us has one its quite complicated for us to debug.

**Comment by jonno85** [comment]
Hi all, it's happening also with the Garmin vivosport.
Thank

**Comment by Stypox** [comment]
Is this still an issue? The notification used to randomly blink even on my phone, but now it does not anymore.

**Comment by myxal** [comment]
I'm not the original reporter, but am seeing the same problem (background player triggering notification spam on the watch) with the following:

- Samsung Galaxy S10e, running Android 9 + OneUI 1.1
- Garmin Connect running on the phone, paired to...
- ... Garmin VivoSmart HR watch
- NewPipe 0.17.3

I'm guessing there's some way to avoid this, as other player with interface in a notification, Bandcamp, doesn't trigger the avalanche, or any notification on the watch whatsoever. I'm willing to collect logs if given instructions on what to collect/how.

**Comment by skontar** [comment]
I have the same problem with Samsung A40 and Samsung S10e when they connect to my car Bluetooth. Does not happen with other Bluetooth devices (and any other players for that matter). Anything I can help with?

**Comment by Stypox** [comment]
Could you test if this is fixed by the apk provided in #3178?

**Comment by brimston3** [comment]
Bit of a non-result, but here are some things I tried.

I used to have this issue w/ garmin connect + fenix 5s. I'd been filtering Newpipe notifications in the Garmin Connect application for a while, but I turned them back on to try to test this bug. I was unable to reproduce the original behavior with the latest version of Connect (4.30) and NewPipe version 0.19.2, nor with the app-debug.apk posted 2020-05-15 in #3178.

Incidentally, Garmin Connect cannot enable or disable notifications from the tested app-debug.apk, so perhaps they are being blocked from the NewPipe side in that version. I can also induce a downloading notification in it by requesting Play over AVRCP when no media app is open, which results in an '[Unknown] [Unknown]' track downloading in the background; the 0.19.2 version has a similar problem but sticks in the playing state. These '[unknown] [unknown]' notifications are not being sent to the smartwatch (yay).

Tested phone was a samsung S7, android 8.0.0, w/ patch 2019-12-01.  
Bluetooth AVRCP triggered w/ bluez 5.50.
